docs/_articles/en/linting.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Linting 
+lang: en
+---
+
+Linting provides errors about malformed code while you edit. VS Code enforces Salesforce’s ESLint rules.
+
+## Prerequisites
+
+[The active LTS release](https://nodejs.org/en/about/releases/) of Node.js built with SSL support.
+
+## Installation
+
+### For a New Project
+
+If you create an SFDX project using the `sfdx force:project:create` command, your project contains a `package.json` file with the ESLint plugin already included.
+
+1. To install the ESLint plugin and other dependencies, run `npm install` in your project directory.
+
+2. Copy the [.eslintrc.json](https://github.com/forcedotcom/salesforcedx-templates/blob/master/src/templates/project/.eslintrc.json) file and save it to your project directory. Configure the ESLint plugin rules as needed.
+
+3. To run linting, you must have components in your project. To start linting, run `npm run lint:lwc`.
+
+### For an Existing Project
+
+1. Verify that your project has a `package.json` with these configurations. If it doesn't, If it doesn't, run `npm install @salesforce/eslint-config-lwc eslint@5 -D`.
```

Comment:
If it doesn't, If it doesn't,

Reply:
I removed

Reply:
K

Reply:
Assume you're re-uploading then and I'll verify that?